examples/README.md

```diff
@@ -0,0 +1,164 @@
+because of its apparent simplicity. There is a catch though: a turtle may only
+turn in place or go in a straight line. So then we have to decompose the circle
+into small lines and turns in order to obtain a regular polygon with just a lot
+of sides so that it looks like a circle.
+
+### Dashed Circle
+
+![dashed_circle](../docs/assets/images/examples/dashed_circle.gif)
+
+**Source Code**: [dashed_circle.rs](dashed_circle.rs)
+
+This one is just a variant of the previous showcasing the ability to raise or
+lower the drawing pen using the `pen_up` and `pen_down` turtle methods.
+
+### Radioactive Triangles
+
+![three-triangles](../docs/assets/images/examples/three-triangles.gif)
+
+**Source Code**: [three-triangles.rs](three-triangles.rs)
+
+Shows how a simple operation like drawing an equilateral triangle can be repeated
+in order to get a more complex result.
+
+### Five Branch Star
+
+![empty_star](../docs/assets/images/examples/empty_star.gif)
+
+**Source Code**: [empty_star.rs](empty_star.rs)
+
+Simple five branch star with an empty middle that colors the background and the
+pen with a thicker pen size as well, using respectively the `set_background_color`,
+`set_pen_color` and `set_pen_size` turtle methods.
+
+## Level: Intermediate
+### Color-filled Squares Spiral
+
+![squares](../docs/assets/images/examples/squares.gif)
+
+**Source Code**: [squares.rs](squares.rs)
+
+This is a repetition of the square drawing, but with added colors and an inner
+colored fill using the `set_fill_color`, `begin_fill` and `end_fill` methods.
+
+### Isometric Nested Cubes
+
+![nestedcubes](../docs/assets/images/examples/nestedcubes.gif)
+
+**Source Code**: [nestedcubes.rs](nestedcubes.rs)
+
+This one is quite interesting: it draws a hexagonal spiral with a regularly
+fading color until it makes it go back to the original one, but it slowly
+reveals a form of visual illusion showing nested concentric three-dimensional
+cubes isometrically projected onto the two-dimensional space of the canvas.
+
+### Randomly-colored Tornado
+
+![coloredridge](../docs/assets/images/examples/coloredridge.gif)
+
+**Source Code**: [coloredridge.rs](coloredridge.rs)
+
+This color frenzy is also nice to look at: the regular shape is generated using
+a gaussian function setting the pen size.
+
+### Heart Emoji
+
+![heart](../docs/assets/images/examples/heart.gif)
+
+**Source Code**: [heart.rs](heart.rs)
+
+Cute heart. The arcs are not as easy to draw as one could imagine at first.
+
+### Rust Logo
+
+![rust](../docs/assets/images/examples/rust.gif)
+
+**Source Code**: [rust.rs](rust.rs)
+
+The - approximate - Rust language logo using this crate!
+
+### Snowman
+
+![snowman](../docs/assets/images/examples/snowman.gif)
+
+**Source Code**: [snowman.rs](snowman.rs)
+
+A nice-looking snowman with all its common features.
+
+### Turtle
+
+![draw_turtle](../docs/assets/images/examples/draw_turtle.gif)
+
+**Source Code**: [draw_turtle.rs](draw_turtle.rs)
+
+A turtle drawn by... a turtle!
```

Comment:
Love the personality you've added to these descriptions! 😁 

(Sorry, I've removed so much of it in my edits 😅)

Reply:
It's fine, don't worry.